pkg/test/istio.io/examples/example.go

```diff
@@ -121,13 +117,23 @@ func (example *Example) Run() {
 		Run(func(ctx framework.TestContext) {
 			kubeEnv, ok := ctx.Environment().(*kube.Environment)
 			if !ok {
-				example.t.Fatalf("test framework unable to get Kubernetes environment")
+				scopes.CI.Errorf("test framework unable to get Kubernetes environment\n")
+				example.t.Fail()
 			}
+
 			for _, step := range example.steps {
+				step.Copy("output/")
 				output, err := step.Run(kubeEnv, example.t)
-				example.t.Log(output)
+
+				//let the step state what it's doing.
```

Comment:
Here and elsewhere, remove all \n from logging statements.

Reply:
Fixed. Thanks.